negotiation/contract.negotiation.protocol.md

```diff
@@ -46,7 +46,7 @@ The CN state machine is transitioned upon receipt and acknowledgement of a messa
 ### Notes
 
 - Concrete wire formats are defined by the protocol binding, e.g. HTTPS.
-- All [Policy](../model/terminology.md#policy) types ([Offer](../model/terminology.md#offer), [Agreement](../model/terminology.md#agreement)) must contain an unique identifier in the form of a URI. GUIDs can also be used in the form of URNs, for instance following the
+- All [Policy](../model/terminology.md#policy) types ([Offer](../model/terminology.md#offer), [Agreement](../model/terminology.md#agreement)) must contain a unique identifier in the form of a URI. GUIDs can also be used in the form of URNs, for instance following the
```

Comment:
probably add a comment somewhere here in the document tht a `target` attribute MUST be present here in the `dspace:dataset->target` (which is different to the `Offer` in the catalog that gets this from the enclosing `dataset` context)